### ISSUE TITLE
Throws.matchesSafely() is not accessible

### ISSUE THREAD

**Issue by fabriciofx** [llorllale/cactoos-matchers#90]
`Throws.matchesSafely()` is not acessible. Let's do it `public`.

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) please, pay attention to this issue

**Comment by llorllale** [comment]
@fabriciofx why?

**Comment by ilyakharlamov** [comment]
@llorllale I guess so that it can be used in other projects.

**Comment by fabriciofx** [comment]
@llorllale Exactly as said by @ilyakharlamov .

**Comment by llorllale** [comment]
@fabriciofx @ilyakharlamov it's not meant to be directly used by user-defined classes.

What am I missing here?

**Comment by fabriciofx** [comment]
@llorllale It's a good question. @ilyakharlamov requested it because (I think) he want to implement some functionality to finish a PR properly. But I've made fast research and discovered we have 9 protected `matchesSafely()` methods and 5 public. What's the reason for that?

**Comment by llorllale** [comment]
@fabriciofx they were probably implemented before teamed/qulice#901 was fixed. I believe a refactor after #75 will fix this.

**Comment by llorllale** [comment]
@ilyakharlamov can we close this?

**Comment by fabriciofx** [comment]
@llorllale Yes. I'll close it.

**Comment by 0crat** [comment]
Job `gh:llorllale/cactoos-matchers#90` is not assigned, can't get performer

**Comment by 0crat** [comment]
This job is not in scope
